File: 20-Lambdas-and-Built-In-Functions/70-triple_and_filter.js
function tripleAndFilter(nums) {
    return nums.filter(n => n % 4 === 0).map(n => n * 3)
}


// === tests ===
console.log(tripleAndFilter([1,2,3,4]))
console.log(tripleAndFilter([6,8,10,12]))

File: 20-Lambdas-and-Built-In-Functions/70-triple_and_filter.py
# === using map and filter ===
def triple_and_filter(nums):
    return map(lambda n: n*3, filter(lambda n: n % 4 == 0, nums))

# === using list comprehension ===
def triple_and_filter(nums):
    return [n * 3 for n in nums if n % 4 == 0]

